.ocamlformat

```diff
@@ -0,0 +1,7 @@
+version = 0.16.0
+profile = conventional
```

Comment:
It is. I added it because not specifying the profile plays a bit weirdly with those who have global OCamlformat files (e.g. in `~/.config/ocamlformat`). If one doesn't specify the profile explicitly, any options set in global config files become the new defaults for project configuration, which can make formatting non-reproducible. Probably not worth worrying about, but I've been bitten by it once or twice.

Reply:
Oh ok, nice, I didn't know about that. Thanks for the explanation!